co-log-core/src/Colog/Core/Action.hs

```diff
@@ -210,7 +210,12 @@ cmap :: (a -> b) -> LogAction m b -> LogAction m a
 cmap f (LogAction action) = LogAction (action . f)
 {-# INLINE cmap #-}
 
--- | Operator version of 'cmap'.
+{- | Operator version of 'cmap'.
+
+>>> logString = LogAction putStrLn
```

Comment:
I think it's better to use `logStringStdout` from `Colog.Core.IO` module

Reply:
This would require a cyclic dependency, since Core.IO depends on Core.Action. Or are you suggesting to reuse the name from Core.IO for clarity?

Reply:
@cronokirby cyclic dependency is not a problem, because `doctest` has it's own `$setup` comment to import other modules. See this module in `relude` for example:

https://github.com/kowainik/relude/blob/9e3aa751851b5a4d4301119c9053275e520fb977/src/Relude/String/Conversion.hs#L62-L64

Even if `Relude` imports `Relude.String.Conversion`, it's still okay to import `Relude` in the comments.

Reply:
Good to know!